libs/header/sync/sync_head.go

```diff
@@ -183,6 +183,9 @@ func (s *Syncer[H]) validate(ctx context.Context, new H) pubsub.ValidationResult
 	return pubsub.ValidationAccept
 }
 
+// TODO(@Wondertan): We should request TrustingPeriod from the network's state params or
```

Comment:
can you put in issue?

Reply:
Will revalidate this first

Reply:
https://github.com/celestiaorg/celestia-node/issues/1857